pandas/tests/frame/methods/test_dtypes.py

```diff
@@ -79,6 +80,24 @@ def test_dtypes_are_correct_after_column_slice(self):
             Series({"a": np.float_, "b": np.float_, "c": np.float_}),
         )
 
+    @pytest.mark.parametrize(
+        "data",
+        [
+            DataFrame(
```

Comment:
Nit: Since the `DataFrame` and `convert_dtypes()` call is the same in these parameters, could you move that into the function?

Reply:
Got it! Thanks!